src/wallet/rpcwallet.cpp

```diff
@@ -2461,14 +2461,15 @@ UniValue z_listunspent(const UniValue& params, bool fHelp)
             "\nResult\n"
             "[                             (array of json object)\n"
             "  {\n"
-            "    \"txid\" : \"txid\",          (string) the transaction id \n"
```

Comment:
Don't remove these spaces; they are necessary for the text to line up in the help output (because the `\"` become `"`).